tests/DatabaseDriverTest.php

```diff
@@ -80,4 +80,15 @@ public function test_all_previous_valid_token_could_revoke()
         $this->assertTrue($this->checkToken($key, $token2));
     }
 
+    public function test_wrong_input_couldnt_revoke_token()
+    {
+        $token = $this->issueToken($key = 'test-key');
+
+        $wrong_token = 'aaa';
```

Comment:
@amirh1374 please use camelCase for varibale naming

Reply:
I will remember that. Would you like me to change it to camelCase and pushing it again?